docs/src/pages/migration-guide.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Migrating to v0.21
+description: How to migrate projects from Astro v0.20.
+---
+
+Starting in v0.21, Astro is now built with [Vite].
+As a result, configurations written in `snowpack.config.mjs` should be moved into `astro.config.mjs`.
+
+```js
+// @ts-check
+
+/** @type {import('astro').AstroUserConfig} */ 
+export default ({
+  renderers: [],
+  vite: {
+    plugins: []
+  }
+})
+```
+
+To learn more about configuring Vite, please visit their [configuration guide](https://vitejs.dev/config/).
+
+
+
+## Aliases
+
+In Astro v0.21, import aliases can be added using `tsconfig.json` or `jsconfig.json`.
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": ["components/*"]
+    }
+  }
+}
+```
+
+These aliases are integrated automatically into [VSCode](https://code.visualstudio.com/docs/languages/jsconfig) and other editors.
+
+
+
+## Aliases (#1749)
+
+In Astro v0.21, import aliases can also be added using the `alias` property.
+
+```js
+export default {
+  vite: {
+    resolve: {
+      alias: {
+        @components: '/src/components',
+      }
+    }
+  }
+}
+```
+
+Note: Always start alias values with `/`. Relative alias starting with `.` will be used as-is and will not be resolved.
+
+
+
+## Passing variables into scripts and styles
+
+In Astro v0.21, variables can be passed from Frontmatter into inline `<style>` and `<script>`.
+
+```astro
+---
+// tick.astro
+const colors = { foregroundColor: "rgb(221 243 228)", backgroundColor: "rgb(24 121 78)" }
+---
+<style define:vars={green}>
+  h-tick {
+    background-color: var(--backgroundColor);
+    border-radius: 50%;
+    color: var(--foregroundColor);
+    height: 15px;
+    width: 15px;
+  }
+</style>
+<h-tick>✓</h-tick>
+```
+
+
+
+## Components in Markdown
+
+In Astro v0.21, Components from any framework can be used within Markdown files.
+
+```md
+---
+Layout: '...'
+setup: | 
+  import MyReactComponent from '../components/MyReactComponent.jsx'
+---
+
+# Hydrating on visibility
+
+<MyReactComponent client:visible>
+
+# Hello world!
+
+</MyReactComponent>
+```
+
+
+
+## Environment variables
+
+In Astro v0.21, environment variables are loaded from `.env` files in your project directory.
+
+```ini
+.env                # loaded in all cases
+.env.local          # loaded in all cases, ignored by git
+.env.[mode]         # only loaded in specified mode
+.env.[mode].local   # only loaded in specified mode, ignored by git
+```
+
+For security purposes, only variables prefixed with `PUBLIC_` are accessible to your code.
+In prior releases, these variables were prefixed with `SNOWPACK_PUBLIC_` and required the `@snowpack/plugin-env` plugin.
```

Comment:
the example below shows `VITE_` prefix, to be honest I don't know what prefix is the right one

Reply:
Thanks! Fixed in 5b1a7747e72590c75de54049d99a00afb4d9dd65